object_tracker/object_tracker.go

```diff
@@ -186,13 +227,41 @@ func (t *myTracker) run(stream gostream.VideoStream, cancelableCtx context.Conte
 	}
 }
 
+func (t *myTracker) trigger() {
+	if t.triggerCancelFunc != nil {
+		t.triggerCancelFunc()
+	}
```

Comment:
I think you will want to add a code block like this to the Close() method as well, to make sure all background threads are closed on the module close